test/test-kitchen.ts

```diff
@@ -8,14 +8,31 @@ import * as pify from 'pify';
 import * as rimraf from 'rimraf';
 import * as tmp from 'tmp';
 
+interface ExecResult {
+  exitCode: number;
+  stdout: string;
+  stderr: string;
+}
+
 const pkg = require('../../package.json');
 
 const rimrafp = pify(rimraf);
 const mkdirp = pify(fs.mkdir);
-const execp = pify(cp.exec);
+const simpleExecp = pify(cp.exec);
 const renamep = pify(fs.rename);
 const ncpp = pify(ncp.ncp);
 
+const execp =
```

Comment:
This deserves a comment. Also, we should probably look for an ecosystem module for execp.